magento/magento2
# Fix issue no touch event on mobile

### Description (*)

Delivery patch for recent fix fotorama js issue touch image in product details page.
Feel free to leave any review or comment any your thoughts


Excellent docs from mozilla
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#improving_scrolling_performance_with_passive_listeners

Logic here
wheel -> passive true cause listener can't block page rendering while the user is scrolling
non wheel (mobile) -> passive false

object options with property passive will pass to event listener at third params

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#37232
### Manual testing scenarios (*)


For image type
1. Apply version to 2.4.6
2. Go to any product details (product has media at least 2 images) on mobile device
3. Try to swipe image
4. Image should able to swipe to next image without problems (Previously before patch can't swipe image on mobile)

Also should Re-test for images-videos mixed case

Further tests
Go click on image to enter fullscreen mode
try next image and close fullscreen (X icon) expect works without problems 

PC browser console no errors related with fotorama

Cross-browsers checklists

- Chrome
- Chrome (mobile/touch-events enabled)
- iOS (Chrome & Safari - latest)
- Chrome Android (latest)
- Firefox
 

### Questions or comments


### Contribution checklist (*)
